docs/monthly-meeting/2022-03.md

```diff
@@ -0,0 +1,126 @@
+# Documentation Community's Team Meeting (March 2022)
+
+- **Date:** 2022-03-07
+- **Time:** [22:00 UTC](https://arewemeetingyet.com/UTC/2022-02-07/22:00/Docs%20WG)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **GitHub issue:** Agenda Planning Issue
+- **Calendar for future meetings:** XXX
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+- name / affiliation / `@github-username`
+- Petr / RedHat, SC / `@encukou`
+- Mariatta / Core Dev / `@mariatta`
+- Pradyun / ??? / `@pradyunsg`
+- CAM / NASA, PEP infra team / `@CAM-Gerlach`
+- James Gerity / random user / `@SnoopJeDi`
+- Julien Palard / Core Dev / `@JulienPalard`
+- Ned / Community member /
+- Adam /
+
+
+## Quick updates - Introductions
+
+- [name=Julien] https://github.com/sphinx-contrib/sphinx-lint
+  - fork of the rstlint.py from CPython, as an independent project
+
+## Reports and celebrations
+
+
+* Shout out to @AA-Turner, Barry, Mariatta and everyone who contributed for getting PEP 676 over the finish line! Its a huge improvement both for us PEP editors, and the Python community at large. — [name=CAM] 
```

Comment:
```suggestion
* Shout out to @AA-Turner, Barry, Mariatta and everyone who contributed for getting PEP 676 over the finish line! Its a huge improvement both for us PEP editors, and the Python community at large. — CAM
```

Reply:
I don't mind either way, but just leaving these [name=...] blocks has basically very little cost, and it's a bunch of busy work to remove it in each meeting's notes. :)

Reply:
Yeah, I initially was going to propose the same but quickly decided it wasn't worth the effort and churn for little practical benefit. I'm sure we all have more important things to do.